apps/portals/src/portal-components/elportal/ELBetaLaunchBanner.tsx

```diff
@@ -15,7 +15,7 @@ const ELBetaLaunchBanner = () => {
       }}
       variant={'info'}
       show={true}
-      title='Welcome to the beta version of the ELITE Portal!'
+      title="Welcome to the beta version of the ELITE Portal!"
```

Comment:
Apologies, a number of small cosmetic changes like this happened when i merged in main. I may be wrong but I think they match the defined linting settings, so I did not revert them.